template/model/catalog-info.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: {{ values.modelName }}
+  annotations:
+    backstage.io/techdocs-ref: dir:{{ values.modelName }}
+  description: '{{ values.modelDescription }}'
+  links:
+    - url: {{ values.modelURL }}
```

Comment:
again, probably outside the scope if this PR specifically, but for me it is in line with the discussion you @johnmcollier and I had alone with @amfred 

if modelURL and modelAPIURL are on Component and possibly API, should we just liet resource and license URL here ? ... fwiw as I'm working through my latest pass at the ODH/RHOAI/Kubeflow mapping to our model catalog, feels like *maybe* resource is where we should focus on download files, licenses, and *maybe* versions ....

but I'm fine punting on this discussion outside of this PR if that is your preference.

Reply:
Yeah, I think keeping things like Download, license links, etc on the Resource probably makes sense.